state-sync/inter-component/test-helpers/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_infallible::RwLock;
+use aptos_types::transaction::{Transaction, WriteSetPayload};
+use aptos_vm::AptosVM;
+use aptosdb::AptosDB;
+use claims::assert_ok;
+use executor_test_helpers::bootstrap_genesis;
+use std::sync::Arc;
+use storage_interface::DbReaderWriter;
+
+pub fn create_database() -> Arc<RwLock<DbReaderWriter>> {
```

Comment:
FYI: I need to look at the details in this PR, but from a quick skim, it seems like overkill to me to create a new create for a single test helper. I'd just copy/paste this 😄

Reply:
Cool, makes things easier :) CLion was giving me shame about the copied code :P